roles/openshift_logging_mux/tasks/main.yaml

```diff
@@ -177,6 +177,48 @@
   check_mode: no
   changed_when: no
 
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
```

Comment:
@nhosoi I don't believe im making myself clear.  I'm not suggesting you move the task or template to a different role but to consolidate the two pvc tasks into one within this role.  Regarding the name I think you need to do something like 'set_facts' for the names in a prior step and then consume them in subsequent tasks